k8s/trillian/Makefile

```diff
@@ -0,0 +1,92 @@
+include ../app.Makefile
+include ../crd.Makefile
+include ../gcloud.Makefile
+include ../var.Makefile
+
+TAG ?= 1.2.1
+APP_DEPLOYER_IMAGE ?= $(REGISTRY)/trillian/deployer:$(TAG)
+NAME ?= trillian-1
+
+ifdef IMAGE_TRILLIAN_LOGSERVER
+  IMAGE_TRILLIAN_LOGSERVER_FIELD = , "IMAGE_TRILLIAN_LOGSERVER": "$(IMAGE_TRILLIAN_LOGSERVER)"
+endif
+
+ifdef IMAGE_TRILLIAN_LOGSIGNER
+  IMAGE_TRILLIAN_LOGSIGNER_FIELD = , "IMAGE_TRILLIAN_LOGSIGNER": "$(IMAGE_TRILLIAN_LOGSIGNER)"
+endif
+
+ifdef IMAGE_MYSQL
+  IMAGE_MYSQL_FIELD = , "IMAGE_MYSQL": "$(IMAGE_MYSQL)"
+endif
+
+ifdef IMAGE_ETCD_OPERATOR
+  IMAGE_ETCD_OPERATOR_FIELD = , "IMAGE_ETCD_OPERATOR": "$(IMAGE_ETCD_OPERATOR)"
+endif
+
+APP_PARAMETERS ?= { \
+  "NAME": "$(NAME)", \
+  "NAMESPACE": "$(NAMESPACE)" \
+  $(IMAGE_TRILLIAN_LOGSERVER_FIELD) \
+  $(IMAGE_TRILLIAN_LOGSIGNER_FIELD) \
+  $(IMAGE_MYSQL_FIELD) \
+  $(IMAGE_ETCD_OPERATOR_FIELD) \
+}
+APP_TEST_PARAMETERS ?= {}
+
+
+app/build:: .build/trillian/db_server \
+            .build/trillian/deployer \
+            .build/trillian/log_server \
+            .build/trillian/log_signer
+
+
+.build/trillian: | .build
+	mkdir -p "$@"
+
+
+.build/trillian/db_server: .build/var/REGISTRY \
+                           .build/var/TAG \
+                           | .build/trillian
+	docker pull gcr.io/trillian-opensource-ci/db_server:$(TAG)
```

Comment:
Could you please take a look at the versioning of images at gcr.io/trillian-opensource-ci? It looks like we have there tags matching the commits, not semantic versions.

Reply:
We have both - there's a tag for every commit to master, as well as a tag for every released version.